src/variables/selectors/index.tsx

```diff
@@ -211,22 +210,8 @@ export const getVariable = (state: AppState, variableID: string): Variable => {
   }
 
   if (!vari.selected.length && vals.length) {
-    try {
+    if (Object.isExtensible(vari.selected)) {
```

Comment:
Let's use `isExtensible` here. Per the error, we know `vari.selected` isn't extensible, but not the precise way it was made not extensible - e.g., `Object.freeze`, `Object.preventExtensions`, `Object.seal`. It's possible for the object not to be _frozen_ but still be unextensible if properties can be deleted.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/isExtensible
https://www.oreilly.com/library/view/javascript-the-definitive/9781449393854/rn01re159.html